### PR TITLE
Multiple Symbols

### PR DESCRIPTION
**Pull request by Ordinary205**

### CODE REVIEW

File: modules/libSceLibcInternal/entry.cpp [hunk 1]
```diff
@@ -93,6 +93,14 @@ EXPORT SYSV_ABI int __NID(strncmp)(const char* str1, const char* str2, size_t si
...
 }
 
+EXPORT SYSV_ABI int __NID(strncpy)(char* dst, char* src, int len ) {
+  return Ok;
+}
+
+EXPORT SYSV_ABI int __NID(strcmp)(char* a, char* b) {
+  return Ok;
+}
+
```
Comment (igor725) [inline modules/libSceLibcInternal/entry.cpp:103]: You can implement those actually, it's not that hard, look at the surrounding functions and do the same thing

**Comment by kr1spy2115** [comment]
Good job with formatting xd

File: modules/libScePad/entry.cpp [hunk 2]
```diff
@@ -260,6 +260,19 @@ EXPORT SYSV_ABI int scePadDeviceClassGetExtendedInformation(int32_t handle, SceP
   return Ok;
 }
 
+EXPORT SYSV_ABI int scePadOpenExt(int userId, int type, int index, int param) {
```
Comment (SysRay) [inline modules/libScePad/entry.cpp:263]: forward to scePadOpen

File: modules/libScePad/entry.cpp [hunk 3]
```diff
@@ -260,6 +260,19 @@ EXPORT SYSV_ABI int scePadDeviceClassGetExtendedInformation(int32_t handle, SceP
...
+  LOG_USE_MODULE(libScePad);
+  LOG_DEBUG(L"todo %S", __FUNCTION__);
+  return Ok;
+}
+
+EXPORT SYSV_ABI int scePadGetExtControllerInformation(int32_t handle, ScePadExtControllerInformation* pInfo) {
```
Comment (SysRay) [inline modules/libScePad/entry.cpp:269]: same here

**Pull request merged**